frontend/packages/component-library/src/cms/section/Section.tsx

```diff
@@ -31,7 +31,7 @@ export interface SectionProps extends HTMLAttributes<HTMLElement> {
   /** Background image */
   backgroundImageUrl?: string;
   /** Vertical padding */
-  padding?: Exclude<ComponentSizeXSToL, 'xs' | 's'>;
+  padding?: Exclude<SpacingNoneToL, 'none' | 'xs' | 's'>;
```

Comment:
I think we can add `SpacingNoneToS` here and update `SpacingNoneToM` to follow `SpacingNoneToL` approach since it looks like it's possible that we have more components/props supporting `SpacingNoneToS` type
smth like this: `type SpacingNoneToS = '' | '' | ''; type SpacingNonetToM = SpacingNoneToS | ''; type SpacingNoneToL = SpacingNoneToM | ''`

Reply:
Great idea! Updated [add SpacingNoneToS type](https://github.com/code-dot-org/code-dot-org/pull/63961/commits/c90840ef57868ed8bb784d0c572fd95b3892fec9) and [exclude SpacingNoneToS](https://github.com/code-dot-org/code-dot-org/pull/63961/commits/1662f7547d2d086d0a6e7f8b1a7dafd6ed2740f6) 🙌